src/components/BottomNav.js

```diff
@@ -40,6 +37,7 @@ export default function FullWidthTabs() {
             aria-label="shopping list"
             component={Link}
             to="/list"
+            style={{ borderRight: '0.1em solid white', padding: '0.5em' }}
```

Comment:
Was there a reason for adding the `padding` here? Only because it gives the two tabs different amounts of padding, but I think they look fine uniform. Love the `borderRight`, really helps to set the two apart!